webapp/components/survey/NodeDefDetails/TaxonProps.js

```diff
@@ -16,10 +16,10 @@ import { useI18n } from '@webapp/store/system'
 import { useSurvey } from '@webapp/store/survey'
 
 import TaxonomyList from '@webapp/views/App/views/Designer/TaxonomyList'
-import TaxonomyComponent from '@webapp/views/App/views/Designer/Taxonomy'
+import TaxonomyDetail from '@webapp/views/App/views/Designer/TaxonomyDetail'
```

Comment:
these two components  go to common survey component

Reply:
I will do in other PR